source/features/lines-changed-per-commit.tsx

```diff
@@ -0,0 +1,52 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+
+const getCommitChanges = async (commit: string): Promise<number[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+            object(expression: "${commit}") {
+                ... on Commit {
+                    additions
+                    deletions
+                    }
+                }
+		}
```

Comment:
Indentation. Don’t wait for my review to apply these changes, make sure your PR is “finalized” before asking for a review, I’m not your autocorrect